style.css

```diff
@@ -1 +1,48 @@
-/* stye in here */
+html {
+    font-size: 10px;
+    font-family: 'Lato', sans-serif;;
+}
+
+.nav-bar {
+    display: flex;
+    flex-direction: row;
```

Comment:
you could use flex-flow: row wrap instead of the two properties flex direction and flex wrap